### PR TITLE
New package: py-neurolab

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-neurolab/package.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+class PyNeurolab(PythonPackage):
+    """Simple and powerfull neural network library for python"""
+
+    homepage = "http://neurolab.googlecode.com/"
+    pypi     = "neurolab/neurolab-0.3.5.tar.gz"
+
+    version('0.3.5', sha256='96ec311988383c63664f3325668f27c30561cf4349e3bc5420665c042a3b9191')
+
+    depends_on('python@2.X:2.Y,3.Z:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-neurolab/package.py]: This line can be removed

File: var/spack/repos/builtin/packages/py-neurolab/package.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    depends_on('py-numpy', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-neurolab/package.py:17]: Can you add a build dep on setuptools? This package currently uses distutils, but once #27798 is merged, we'll be using pip, and pip replaces all calls to distutils with setuptools.

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-neurolab/package.py (hunk 2 above)
Comment (iarspider) [reply var/spack/repos/builtin/packages/py-neurolab/package.py:17]: It fails with setuptools:
```
1 error found in build log:
     26    Warning: 'classifiers' should be a list, got type 'tuple'
     27    usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     28       or: setup.py --help [cmd1 cmd2 ...]
     29       or: setup.py --help-commands
     30       or: setup.py cmd --help
     31
  >> 32    error: option --single-version-externally-managed not recognized
```

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-neurolab/package.py:17]: Oh right. Just add it and comment it out. If you comment on #27798 after this is merged I'll rebase and fix it. Same for any other packages I've mentioned this about.

Comment (iarspider) [reply var/spack/repos/builtin/packages/py-neurolab/package.py:17]: @adamjstewart done

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**